src/GraphQLParser/Visitors/SDLWriter.cs

```diff
@@ -482,21 +572,48 @@ public override async ValueTask VisitInterfaceTypeDefinition(GraphQLInterfaceTyp
             await Visit(interfaceTypeDefinition.Name, context).ConfigureAwait(false);
             await VisitInterfaces(interfaceTypeDefinition, context).ConfigureAwait(false);
             await VisitDirectives(interfaceTypeDefinition, context).ConfigureAwait(false);
-            await context.WriteLine().ConfigureAwait(false);
-            await context.Write("{").ConfigureAwait(false);
-            await context.WriteLine().ConfigureAwait(false);
 
             if (interfaceTypeDefinition.Fields?.Count > 0)
             {
+                await context.WriteLine().ConfigureAwait(false);
```

Comment:
Three lines (and two more below) were moved under `if` to handle empty interfaces.